pkgs/development/libraries/readline/6.3.nix

```
@@ -34,6 +34,8 @@ stdenv.mkDerivation {
      in
        import ./readline-6.3-patches.nix patch);
 
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=implicit-function-declaration";
```

```suggestion
  env = lib.optionalAttrs stdenv.cc.isClang {
    NIX_CFLAGS_COMPILE = "-Wno-error=implicit-function-declaration";
  };
```
nit: using an attribute set won't change the derivation for `!isClang`.

---

thanks!